src/test/scala/io/pravega/connectors/spark/PravegaSinkSuite.scala

```diff
@@ -1,537 +1,537 @@
-          withSelectExpr = s"value as ${EVENT_ATTRIBUTE_NAME}")
-        input.addData("1", "2", "3", "4", "5")
-        writer.processAllAvailable()
-      }
-    } finally {
-      writer.stop()
-    }
-    assert(PravegaTestUtils.exceptionString(ex).toLowerCase(Locale.ROOT).contains("job aborted"))
-  }
-
-  private val streamNumber = new AtomicInteger(0)
-
-  private def newStreamName(): String = s"sinkstream${streamNumber.getAndIncrement()}"
-
-  private def createPravegaReader(streamName: String): DataFrame = {
-    spark.read
-      .format(SOURCE_PROVIDER_NAME)
-      .option(CONTROLLER_OPTION_KEY, testUtils.controllerUri)
-      .option(SCOPE_OPTION_KEY, testUtils.scope)
-      .option(STREAM_OPTION_KEY, streamName)
-      .option(START_STREAM_CUT_OPTION_KEY, STREAM_CUT_EARLIEST)
-      .option(END_STREAM_CUT_OPTION_KEY, STREAM_CUT_LATEST)
-      .load()
-  }
-
-  private def createPravegaWriter(
-      input: DataFrame,
-      withStreamName: Option[String] = None,
-      withOutputMode: Option[OutputMode] = None,
-      withOptions: Map[String, String] = Map[String, String]())
-      (withSelectExpr: String*): StreamingQuery = {
-    var stream: DataStreamWriter[Row] = null
-    withTempDir { checkpointDir =>
-      var df = input.toDF()
-      if (withSelectExpr.nonEmpty) {
-        df = df.selectExpr(withSelectExpr: _*)
-      }
-      stream = df.writeStream
-        .format(SOURCE_PROVIDER_NAME)
-        .option("checkpointLocation", checkpointDir.getCanonicalPath)
-        .option(CONTROLLER_OPTION_KEY, testUtils.controllerUri)
-        .option(SCOPE_OPTION_KEY, testUtils.scope)
-        .queryName("pravegaStream")
-      withStreamName.foreach(stream.option(STREAM_OPTION_KEY, _))
-      withOutputMode.foreach(stream.outputMode(_))
-      withOptions.foreach(opt => stream.option(opt._1, opt._2))
-    }
-    stream.start()
-  }
-}
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.connectors.spark
+
+import java.util.Locale
+import java.util.concurrent.atomic.AtomicInteger
+
+import io.pravega.connectors.spark.PravegaReader._
+import io.pravega.connectors.spark.PravegaSourceProvider._
+import org.apache.spark.SparkException
+import org.apache.spark.sql._
+import org.apache.spark.sql.execution.streaming.MemoryStream
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.streaming._
+import org.apache.spark.sql.test.SharedSparkSession
+import org.scalatest.concurrent.PatienceConfiguration.Timeout
+import org.scalatest.time.Span
+import org.scalatest.time.SpanSugar._
+
+class PravegaSinkSuite extends StreamTest with SharedSparkSession with PravegaTest {

```

Comment:
I'm seeing the following lines in the output of this test.
```
null
()
```
Perhaps there is some debug code somewhere that is writing to System.out. This should be removed or changed to use log.debug.